xmodule/templates/html/raw.yaml

```diff
@@ -3,11 +3,7 @@ metadata:
     display_name: Raw HTML
     editor: raw
 data: |
-      <p>This template is similar to the Text template. The only difference is 
-      that this template opens in the Raw HTML editor rather than in the Visual
-      editor.</p>
-      <p>The Raw HTML editor saves your HTML exactly as you enter it.
-      You can switch to the Visual editor by clicking the Settings tab and 
-      changing the Editor setting to Visual. Note, however, that some of your 
-      HTML may be modified when you save the component if you switch to the 
-      Visual editor.</p>
+      <p>This is a Raw HTML editor that saves your HTML exactly as you enter it.
```

Comment:
Looks fine, and certainly good enough to keep. You might consider shortening the first couple of sentences to  "<p>This Raw HTML editor saves your HTML verbatim: it gives you maximum flexibility, but will save and render HTML tags as is, even if malformed. There is no way ..."